SConstruct

```diff
@@ -55,7 +55,10 @@ def get_version():
         return version_file.read().rstrip()
 
 DAOS_VERSION = get_version()
-API_VERSION = "0.9.0"
+API_VERSION = "1.0.0"
```

Comment:
It seems like nobody would miss having to update the above line and the below lines at the same time, but it would be nice to derive the above from the below.